runtime/src/accounts_db.rs

```diff
@@ -1296,13 +1305,19 @@ impl AccountsDB {
         hasher.result()
     }
 
+    pub fn include_owner_in_hash(slot: Slot) -> bool {
+        slot >= 14_000_000
```

Comment:
Can you please comment here that this is a testnet slot index.   Unfortunate that we don't have the operating mode this deep in accounts_db

Reply:
sure!